packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1525,6 +1526,13 @@ void main() {
         expect(metrics, hasLength(1));
     }
   }, skip: kIsWeb && !isCanvasKit); // [intended] Browsers seem to always round font/glyph metrics.
+
+  test('TextPainter dispatches memory events', () async {
+    await expectLater(
+      await memoryEvents(() => TextPainter().dispose(), TextPainter),
+      areCreateAndDispose,
```

Comment:
@polina-c Probably an unfortunate typo in `leak_tracker_flutter_testing` but shouldn't this be named as `areCreatedAndDisposed` ?

Reply:
nope
the name is about events
the name states that the events are about creation and disposal, not that they are created and disposed

Reply:
I see. That indeed makes more sense.